vendor/pacman/js/game.js

```diff
@@ -23,52 +23,81 @@ var TIME_FRUITS = 0;
 
 var HELP_DELAY = 1500;
 var HELP_TIMER = -1;
-			
-function blinkHelp() { 
-	if ( $('.help-button').attr("class").indexOf("yo") > -1 ) { 
+
+let gameOver = new Event("gameOver");
+
+window.onload = function() {
+   document.getElementById("panel").addEventListener("gameOver", scorePopUp);
```

Comment:
Since `GAMEOVER` is a global variable (not declared with `var`), can you trigger on GAMEOVER being set to true (or `SCORE` being reset to zero)?

You might be able to do this by injecting a function into an event handler on the window (timer or keypress), which would avoid needing to edit this file.

Alternatively, you could inject an extra javascript file into the frame which redefines one of the globals like the `gameover` function. e.g.:

```javascript
window.onload(() => {
  let origGameover = gameover;
  gameover = () => {
    orig_gameover();
    scorePopUp();
  }
}
```

Reply:
The last approach worked, needed to add some extra event listeners to make sure everything was loaded correctly first - a8dbb82